tests/pos/open-selftype.scala

```diff
@@ -0,0 +1,8 @@
+
+trait X { self: Y => } // error: missing requirement: self type Y & X of trait X does not conform to self type Z of required trait Y
```

Comment:
Same here. There is no error anymore.

Reply:
True. But we do leave the old errors in anyway if a neg test gets moved to pos as a documentation what went wrong before. That's done for many other pos tests as well.

Reply:
Ah OK. I did not know that. All good, then.